ratpack-micrometer-metrics/src/main/java/ratpack/micrometer/metrics/internal/DefaultBlockingExecTimingInterceptor.java

```diff
@@ -0,0 +1,44 @@
+package ratpack.micrometer.metrics.internal;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Timer;
+import ratpack.exec.ExecInterceptor;
+import ratpack.exec.Execution;
+import ratpack.func.Block;
+import ratpack.http.Request;
+import ratpack.micrometer.metrics.BlockingExecTimingInterceptor;
+import ratpack.micrometer.metrics.MicrometerMetricsConfig;
+
+import java.util.Optional;
+
+public class DefaultBlockingExecTimingInterceptor implements BlockingExecTimingInterceptor {
+  private final MicrometerMetricsConfig config;
+  private final MeterRegistry meterRegistry;
+
+  DefaultBlockingExecTimingInterceptor(MicrometerMetricsConfig config, MeterRegistry meterRegistry) {
+    this.config = config;
+    this.meterRegistry = meterRegistry;
+  }
+
+  @Override
+  public void intercept(Execution execution, ExecInterceptor.ExecType type, Block executionSegment) throws Exception {
+    if (type == ExecInterceptor.ExecType.BLOCKING) {
+      Optional<Request> requestOpt = execution.maybeGet(Request.class);
+      if (requestOpt.isPresent()) {
+        Request request = requestOpt.get();
+        Timer.Sample sample = Timer.start(meterRegistry);
+        try {
+          executionSegment.execute();
+          sample.stop(meterRegistry.timer("http.blocking.execution",
```

Comment:
Side comment - one of the biggest things we are missing is a central place that lists all these measurement names and what they represent (docs perhaps). I also wonder if we should make this pluggable in some way (though that could make this overly-complex). 
Are there any standards out there metrics naming? (like CNCF or anything?)

Reply:
By pluggable do you mean that the user can configure the names to be something else? If so, this is already possible with `MeterFilter`. This interface was built with the idea that users could change names, add/remove/change tags, and assign different base units and description text. There's a config item in `MicrometerMertricsConfig` here that allows Ratpack users to add `MeterFilters`.

There is no open standard for metrics naming. Some notes however:
1. Micrometer uses dot separated names by convention, and each registry implementation is packed with a name normalizer that transforms its style to something the monitoring system expects (e.g. `http.server.requests` would appear as a set of timeseries prefixed with `http_server_requests_seconds` in Prometheus). The idioms of each monitoring system are mutually exclusive, so it's best to follow the dot convention in frameworks and let Micrometer do the dirty work.
2. When possible, this PR follows Spring Boot metrics names, which were in turn copied in Micronaut, etc. So a sort of informal standard is in play already.